data/presets/presets/playground/balance_beam.json

```diff
@@ -1,5 +1,17 @@
 {
     "icon": "maki-playground",
+     "fields": [
+        "playground/theme",
```

Comment:
Tip: Instead of repeating the fields for each playground preset you can set the `fields` and `moreFields` of `_playground.json` and iD will inherit them automatically. You can still override these properties in individual files if you want.

If this doesn't make sense then don't worry about it.